test/spec/components/test/context.test.jsx

```diff
@@ -15,7 +15,7 @@ describe('<TestContext />', () => {
       wrapper,
       ctx: wrapper.find(TestContext),
       ctxItems: wrapper.find('.test-context-item'),
-      img: wrapper.find('.test-image-link'),
+      img: wrapper.find('.test-image'),
```

Comment:
Its ok to change `img` to find `.test-image` but I still need to assert that the `.test-image-link` exits for regular images. I'd just add another property, `imgLink: wrapper.find('.test-image-link')` and then update the image tests to add an assertion for this.